### ISSUE TITLE
Questions about input

### ISSUE THREAD

**Issue by chengcheng8632** [facebookresearch/mvfst-rl#22]
Dear author:
        I'm sorry to disturb you again. In the paper, our input is a 21-dimensional state space, and the value of reward is not used as an input. But in polybeast.py in the train folder, "core_input = torch.cat ([x, clipped_reward], dim = -1)", it seems that the reward is used as input. I ask is that right? What is the purpose of this? Looking forward to your answer. Thank you very much?

**Comment by odelalleau** [comment]
Hi @chengcheng8632, this may indeed not be entirely clear from Appendix A.1 in the paper, but it is mentioned in the "Model" paragraph of Section 4: "_The extracted features and the reward are fed into a single-layer LSTM"_
